keyboards/boardsource/3x4/keymaps/default/keymap.c

```diff
@@ -0,0 +1,25 @@
+
+#include QMK_KEYBOARD_H
+
+#define _MAIN 0
+#define _RAISE 1
+#define _LOWER 2
```

Comment:
In general, we like to use an enum for the layers, as it makes it easier to manage, long term 

```suggestion
enum layers {
    _MAIN,
    _RAISE,
    _LOWER,
);
```